pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -1510,167 +1506,3 @@ func (c *syncingMockCache) UpdateNodeNameToInfoMap(infoMap map[string]*scheduler
 	})
 	return err
 }
-
-// TestCacheInvalidationRace tests that equivalence cache invalidation is correctly
-// handled when an invalidation event happens early in a scheduling cycle. Specifically, the event
-// occurs after schedulercache is snapshotted and before equivalence cache lock is acquired.
-func TestCacheInvalidationRace(t *testing.T) {
```

Comment:
I enjoyed writing this test. Can you also delete `syncingMockCache`?

Reply:
Haha, will do